### ISSUE TITLE
Can u share the training loss curve?

### ISSUE THREAD

**Issue by lucasjinreal** [BAAI-DCAI/Bunny#17]
Can u share the training loss curve?

**Comment by Isaachhh** [comment]
You can find the loss log in `trainer_state.json` in the related HuggingFace repo of each model.
For example, https://huggingface.co/BAAI/bunny-phi-2-siglip-lora/blob/main/trainer_state.json
